project/ScalacOptions.scala

```diff
@@ -57,9 +59,7 @@ object Scalac {
     // "-Ywarn-unused:privates", // Warn if a private member is unused.
     "-Ywarn-value-discard", // Warn when non-Unit expression results are unused.
     "-Xmacro-settings:show-coder-fallback=true",
-    "-Ydelambdafy:inline", // Set the strategy used for translating lambdas into JVM code to "inline"
-    "-Ybackend-parallelism",
-    "8"
```

Comment:
This is parallelism 8, not JDK version 8 right?

Reply:
yeah that is referring to parallelism. It shouldn't be removed actually (I was experiment other stuff). Added it back.